wp-content/themes/access/gulpfile.babel.js

```diff
@@ -376,7 +376,26 @@ gulp.task('svgs:list', () => gulp.src([
 
 gulp.task('svgs:purge', () => gulp.src('assets/svg/icons.svg')
   .pipe(through.obj((chunk, encoding, callback) => {
-    console.dir(list);
+     let iconsInUse = [
+      'icon-card-work-v2',
+      'icon-card-family-services-v2',
+      'icon-card-people-with-disabilities-v2',
+      'icon-card-PROGRAMS-v2',
+      'icon-card-cash-expenses-v2',
+      'icon-cash-expenses-v2',
```

Comment:
<img width="814" alt="Screen Shot 2020-09-25 at 3 54 13 PM" src="https://user-images.githubusercontent.com/14843767/94311066-a178d400-ff48-11ea-9957-d2c423808b70.png">
Looking into why the icon is not showing the dollar sign although is being pass to the `whitelist:`